data.json

```diff
@@ -48,7 +48,7 @@
     "name": "CircleCI",
     "difficulty": "medium",
     "description": "Participate in #Orbtoberfest with CircleCI and claim limited edition swag! 1 - 3 pull requests: Limited-Edition Sticker, 4+ pull requests: Limited-Edition T-shirt",
-    "reference": "https://hacktoberfest.circleci.com/#/" ,
```

Comment:
Theses changes are out of the scope of this PR, can you please revert them?

Reply:
Reverted the changes caused by my linter. Sorry!